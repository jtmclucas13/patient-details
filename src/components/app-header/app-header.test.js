import React from 'react';
import { shallow } from 'enzyme';
import AppHeader from './app-header';

describe('component AppHeader', () => {
    test('renders as expected', () => {
        const wrapper = shallow(<AppHeader />);

        expect(wrapper).toMatchSnapshot();
    });
});
