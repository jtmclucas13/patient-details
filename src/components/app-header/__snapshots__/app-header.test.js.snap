// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component AppHeader renders as expected 1`] = `
<h1
  className="app-header"
>
  Patient Details Hub
</h1>
`;
